tests/functional/fs/test_fs_statistics.py

```diff
@@ -0,0 +1,171 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import time
+import pytest
+
+from core.test_run import TestRun
+from test_tools.disk_utils import Filesystem
+from test_utils.os_utils import sync
+from test_tools.fs_utils import create_directory
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_tools.fio.fio_result import FioResult
+from test_utils.size import Size, Unit
+from api.iotrace_stats_parser import (
+    parse_fs_stats,
+    DirectoryTraceStatistics,
+    FileTraceStatistics,
+    ExtensionTraceStatistics,
+)
+
+
+@pytest.fixture(params=Filesystem)
+def fs(request):
+    file_system = request.param
+    mountpoint = "/mnt"
+    disk = TestRun.dut.disks[0]
+
+    with TestRun.step(f"Create file system {file_system}"):
+        disk.create_filesystem(file_system)
+
+    with TestRun.step("Mount device"):
+        disk.mount(mountpoint)
+
+    yield disk, mountpoint
+
+    with TestRun.step("Unmount device"):
+        disk.unmount()
+
+
+def test_fs_operations(fs):
```

Comment:
I don't think that giving type hints for test functions is necessary - afterall they're not reused anywhere. I even think that it may be a bit misleading to do that in pytest tests, because arguments are expected to be fixtures. What do you think?